src/__tests__/unit/easy.eventOverlap.spec.ts

```diff
@@ -1,37 +1,186 @@
-import { Event } from '../../types';
+import { Event, EventForm } from '../../types';
 import {
   convertEventToDateRange,
   findOverlappingEvents,
   isOverlapping,
-  parseDateTime,
 } from '../../utils/eventOverlap';
 
-describe('parseDateTime', () => {
-  it('2024-07-01 14:30을 정확한 Date 객체로 변환한다', () => {});
+describe('convertEventToDateRange', () => {
+  const sampleEventForm: EventForm = {
+    title: '테스트 이벤트',
+    description: '설명',
+    location: '장소',
+    date: '2024-07-01',
+    startTime: '14:30',
+    endTime: '15:30',
+    category: 'meeting',
+    repeat: {
+      type: 'none',
+      interval: 1,
+    },
+    notificationTime: 30,
+  };
 
-  it('잘못된 날짜 형식에 대해 Invalid Date를 반환한다', () => {});
+  it('일반적인 이벤트를 올바른 시작 및 종료 시간을 가진 객체로 변환한다', () => {
+    const result = convertEventToDateRange(sampleEventForm);
 
-  it('잘못된 시간 형식에 대해 Invalid Date를 반환한다', () => {});
+    expect(result.start).toBeInstanceOf(Date);
+    expect(result.end).toBeInstanceOf(Date);
+    expect(result.start.getHours()).toBe(14);
+    expect(result.start.getMinutes()).toBe(30);
+    expect(result.end.getHours()).toBe(15);
+    expect(result.end.getMinutes()).toBe(30);
+  });
 
-  it('날짜 문자열이 비어있을 때 Invalid Date를 반환한다', () => {});
-});
+  it('잘못된 날짜 형식의 이벤트에 대해 Invalid Date를 반환한다', () => {
+    const invalidDateEvent: EventForm = {
+      ...sampleEventForm,
+      date: '2024-13-01', // 잘못된 월
+    };
 
-describe('convertEventToDateRange', () => {
-  it('일반적인 이벤트를 올바른 시작 및 종료 시간을 가진 객체로 변환한다', () => {});
+    const result = convertEventToDateRange(invalidDateEvent);
+    expect(result.start.toString()).toBe('Invalid Date');
+    expect(result.end.toString()).toBe('Invalid Date');
```

Comment:
마찬가지로
```suggestion
    expect(result).toEqual({
      start: new Date(NaN),
      end: new Date(NaN),
    });
```

이런식으로 표현해줄 수 있답니다!